packages/repository/src/relations/relation.types.ts

```diff
@@ -68,6 +68,29 @@ export interface HasManyDefinition extends RelationDefinitionBase {
   keyTo?: string;
 }
 
+export interface HasManyThroughDefinition extends HasManyDefinition {
```

Comment:
Please add tsdoc comment for this interface, include a note that it's experimental and may change in backward-incompatible ways between semver-minor releases. This should hopefully allow us to iterate on the best format of the relation definition.

Reply:
Sounds good! I updated the comments and updated the interface to use the nested object format.